it/it-projects/testing/xoo-sample-new-coverage-v2/src/main/xoo/sample/Sample.xoo

```diff
@@ -1,14 +1,14 @@
 package sample;
 
 public class Sample {
-	
+
 	public Sample(int i) {
 		int j = i++;
 	}
 	
 	private String myMethod() {
-		if (foo == bar) {
-			return "hello";
+		if (foo2 == bar2) {
```

Comment:
Yes, otherwise the code is the same than version 1, and so new coverage is not detected